test/functionalTest/cases/1080_batch_query_context/batch_query_context_several_filters.test

```diff
@@ -166,20 +166,12 @@ payload='{
     }
   ],
   "attributes": [ ],
-  "scopes": [
```

Comment:
The scope -> expression changes in existing .test may be wrongly interpreted as a rupture in backward compatibility rupture. However, note that scope functionaly was never part of the stable NGSIv2 spec (an also warned in NGSIv2 implementation notes and NGSIv2 walkthrough as a experimental feature).

Thus, this change is safe.

NTC (only informative)